src/main/java/com/blamejared/contenttweaker/blocks/render/BlockRenderTypeCollection.java

```diff
@@ -0,0 +1,28 @@
+package com.blamejared.contenttweaker.blocks.render;
+
+import com.blamejared.contenttweaker.api.blocks.IIsCoTBlock;
+import net.minecraft.util.Util;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.EnumMap;
+import java.util.Map;
+
+public class BlockRenderTypeCollection {
+    private static final Map<BlockRenderType, Collection<IIsCoTBlock>> blockRenderTypeMap = Util.make(new EnumMap<>(BlockRenderType.class), (map) -> {
+        for (BlockRenderType value : BlockRenderType.values()) {
+            map.put(value, new ArrayList<>());
+        }
+    });
+
+    @OnlyIn(Dist.CLIENT)
+    public static void registerAllRenderTypeRules() {
```

Comment:
```suggestion
    public static void registerAllRenderTypeRules() {
```

Sorry for the confusion, after talking with SkySom again, this should not be needed here